src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ActionPlanService.kt

```diff
@@ -100,9 +100,9 @@ class ActionPlanService(
       ?: throw EntityNotFoundException("action plan not found [id=$id]")
   }
 
-  fun getActionPlanByReferral(referralId: UUID): ActionPlan {
+  fun getActionPlanByReferral(referralId: UUID): List<ActionPlan> {
```

Comment:
Old method that was no longer being used, so updated to work with multiple action plans.

Reply:
or if we use the suggested repository method it would be `getApprovedActionPlansByReferral`